Merge Feature/car 67 with Master

oscarhjern opened the pull request.

oscarhjern committed:
#67 Implement turnInPlace()

Implements turn in place tp, turn the car on it spot

oscarhjern committed:
#67 Update comments

Removes code for testing and structures the code in a nice way. Refers to #62

oscarhjern committed:
#67 Implement drive around obsticle

Change our method from wenseday from turnInPlace() to Dimitrios example-method and implement method that allows car to drive around an obsticle (using the new turnInPlace-method

majawennroth committed:
#67 Improve the function of going around an obstical

oscarhjern committed:
#67 Updated obsticle avoidence

oscarhjern committed:
#67 Update avoid obsticle method

oscarhjern committed:
#67 Add missing 1

oscarhjern committed:
#67 Update method

oscarhjern committed:
#67 Updated avoid obsticle

oscarhjern committed:
67 Updated avoid obsticle

oscarhjern committed:
#67 remove driveAroundObsticle-method

oscarhjern committed:
#67 Implements new driveAroundOsticle method

Refers to #65

oscarhjern committed:
#67 Update driveAroundObsticle

oscarhjern committed:
#67 Update TURN_SPEES

majawennroth committed:
#67 Make the car drive around and obstical.

The car does do what its suppesed to but only when it 'feels' like doing it...

majawennroth committed:
#67 Implement an almost always coopperative drive around obstical method.

majawennroth committed:
#67 improve the obsticale avoidence/going around the obstical.

majawennroth committed:
Implement another way of going around an obsical

Refrence #61

oscarhjern committed:
#61 Implement new OA-method

majawennroth committed:
Add changes to the testing code

oscarhjern committed:
#61 Updated OA-method

Have two different versions of the same method. One is if, in while-loop, in for-loop and one is if, in do while-loop

majawennroth committed:
Improve OA

majawennroth committed:
Make the same implementation for going around the obstical but on the left side.

majawennroth committed:
Improved OA

oscarhjern committed:
Merge pull request #88 from DIT112-V20/testOA

Merge Test oa with feature/car-67

oscarhjern committed:
#67 Cleans up the code

Refers to #61. Fixes the merge from test OA and cleans up the code by changing numbers to const int:s, changing name of methods, removing unnecisary lines of code.

oscarhjern committed:
#67 Change distance front.

johann-tam commented on car.ino:
```diff
@@ -0,0 +1,184 @@
+  }
+}
+
+void driveAroundObstical(){
```
Typo **Obstacle**

johann-tam reviewed (changes requested):
Please change Obstical to Obstacle

and merge master into your branch before to get the CI

oscarhjern committed:
Update car.ino

#67 Spelled 'obstacle' correctly ;)

oscarhjern committed:
Master into feature/car-67

# Conflicts:
#	arduinoRemote/.gradle/5.6.4/fileHashes/fileHashes.bin
#	arduinoRemote/.gradle/5.6.4/fileHashes/fileHashes.lock
#	arduinoRemote/.gradle/buildOutputCleanup/buildOutputCleanup.lock
#	arduinoRemote/.gradle/buildOutputCleanup/outputFiles.bin
#	arduinoRemote/local.properties
#	car/car.ino

oscarhjern committed:
#67 Solved merge conflicts locally

Refers to #61. Also added CI to branch feature/car-67.
